File: Models/Profile/DestinyProfileTransitoryComponent.cs
namespace bulkybook.Models;

public class DestinyProfileTransitoryComponent
{
    public List<DestinyProfileTransitoryPartyMember> partyMembers {get;set;}
    public DestinyProfileTransitoryCurrentActivity currentActivity {get;set;}
    public DestinyProfileTransitoryJoinability joinability {get;set;}
    public List<DestinyProfileTransitoryTrackingEntry> tracking {get;set;}
    /*  The hash identifier for the DestinyDestinationDefinition of the last location you were orbiting when in orbit.
Type: uint32
Mapped to Manifest Database Definition: Destiny.Definitions.DestinyDestinationDefinition  */
    public UInt32 lastOrbitedDestinationHash {get;set;}
}

File: Scripts/Models/DestinyProfileTransitoryComponent.ts
import { DestinyProfileTransitoryPartyMember } from "./DestinyProfileTransitoryPartyMember";
import { DestinyProfileTransitoryCurrentActivity } from "./DestinyProfileTransitoryCurrentActivity";
import { DestinyProfileTransitoryJoinability } from "./DestinyProfileTransitoryJoinability";
import { DestinyProfileTransitoryTrackingEntry } from "./DestinyProfileTransitoryTrackingEntry";
export interface DestinyProfileTransitoryComponent
{
    partyMembers: Array<DestinyProfileTransitoryPartyMember>;
    currentActivity: DestinyProfileTransitoryCurrentActivity;
    joinability: DestinyProfileTransitoryJoinability;
    tracking: Array<DestinyProfileTransitoryTrackingEntry>;
    /*  The hash identifier for the DestinyDestinationDefinition of the last location you were orbiting when in orbit.
Type: uint32
Mapped to Manifest Database Definition: Destiny.Definitions.DestinyDestinationDefinition  */
    lastOrbitedDestinationHash: number;
}